addons/azureSets/azuresets.lua

```diff
@@ -68,19 +68,36 @@ windower.register_event('login', initialize)
 
 windower.register_event('job change', initialize:cond(function(job) return job == 16 end))
 
-function set_spells(spellset)
+function set_spells(spellset, setmode)
     if windower.ffxi.get_player()['main_job_id'] ~= 16 --[[and windower.ffxi.get_player()['sub_job_id'] ~= 16]] then return nil end
     if settings.spellsets[spellset] == nil then return end
-    if settings.spellsets[spellset]:equals(get_current_spellset()) then
+    if is_spellset_equipped(settings.spellsets[spellset]) then
         error(spellset..' was already equipped.')
```

Comment:
I know this isn't your code, but I wouldn't error on this, just print a warning and abort. If you're already messing around in this area, may as well adjust this :) But up to you.